main/group_vars/all

```diff
@@ -8,5 +7,4 @@ cosmovisor_version: "v1.3.0"
 sync_node: ""
 sync_node_comm_user: "root"
 slack_hook: ""
-trust_rpc1: "rpc-mainnet.commercio.network:80"
-trust_rpc2: "rpc2-mainnet.commercio.network:80"
+rpc_servers: "{{ trust_rpc1 }},{{ trust_rpc2 }}"
```

Comment:
I think this can't work: `trust_rpc1` and `trust_rpc2` get the value in a subsequent task